user_data/strategies/Strategy002.py

```diff
@@ -165,15 +172,54 @@ def confirm_trade_entry(self, pair: str, order_type: str, amount: float, rate: f
         :return bool: When True is returned, then the buy-order is placed on the exchange.
             False aborts the process
         """
-        print("............................................................................"+str(current_time))
+        print("confirm_trade_entry: BUY --- "+str(current_time))
+
         mode = "test"
         coin = pair.split("/")[0]
         brain = "Freq_" + self.__class__.__name__
         if Config.IS_BACKTEST:
-            back_test(current_time, coin, brain)
-        elif Config.IS_EXECUTION:
-            execute(mode, coin, brain)
-        
+            self.controller.back_test(current_time, coin, brain)
+        else:
+            self.controller.execute(mode, coin, brain)
         return True
 
+    def confirm_trade_exit(self, pair: str, trade: Trade, order_type: str, amount: float,
```

Comment:
this already exists in develop, please merge develop in to your branch
we shouldn't see this change on the pr

Reply:
I did merge it but still the same

Reply:
doesn't make sense, something is wrong in the conflict resolution - create a pr with only your changes